test/fitnesseMain/FitNesseMainTest.java

```diff
@@ -112,13 +114,25 @@ public void runningCommandWithNonExistentAddressResultsInError() throws Exceptio
     String[] args = {"-o", "-a", "user:pwd", "-c", "user:pwd:/FitNesse.NonExistentTestCase?test"};
     Arguments arguments = new Arguments(args);
     try {
-        Integer exitCode = new FitNesseMain().launchFitNesse(arguments);
-    } catch (Exception e){
-        assertEquals("error loading page: 404", e.getMessage());
-        throw e;
+      Integer exitCode = new FitNesseMain().launchFitNesse(arguments);
+    } catch (Exception e) {
+      assertEquals("error loading page: 404", e.getMessage());
+      throw e;
     }
   }
 
+  @Test
+  public void localhostOnlyFlagResultsInConnectableFitnesseOnLocalHost() throws Exception {
```

Comment:
Locally, a `./gradlew test` now fails on many tests. Does it work for you?

I believe this test keeps a server running. Is that indeed the case? If so, please ensure it is cleaned up after the test completes...

Reply:
I have no failing tests... May be OS dependent.
Let me see if I can force close the Fitnesse process

Reply:
The `Shutdown` class will do it.
If I add a call to that (in a finally block) all tests pass...

Reply:
Since I actually have the code available, let me just push it.

Reply:
Changed the test to call the shutdown responder. Could you verify if that prevents the other tests from failing?

Reply:
You were faster than I expected. Testing now....